paasta_tools/cli/cmds/validate.py

```diff
@@ -317,13 +328,27 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
-def validate_tron(service_path):
+def validate_tron(service_path, verbose=False):
     soa_dir, service = path_to_soa_dir_service(service_path)
     returncode = True
 
     for cluster in list_tron_clusters(service, soa_dir):
         if not validate_tron_namespace(service, cluster, soa_dir):
             returncode = False
+        elif verbose:
```

Comment:
thoughts on adding a comment here to explain why this is in an else block? to someone quickly skimming this, this might look a little weird unless they realise that we're doing this to ensure that we're just parsing valid cron strings